lib/iris/_concatenate.py

```diff
@@ -321,6 +321,7 @@ def __init__(self, cube):
         self.dim_metadata = []
         self.ndim = cube.ndim
         self.scalar_coords = []
+        self._cell_measures_and_dims = cube._cell_measures_and_dims
```

Comment:
@marqh `_cell_measures_and_dims` -> `cell_measures_and_dims`

No need to make this private, the whole class is private, so let's follow the previous naming convention for the class attributes.